src/index.js

```diff
@@ -384,7 +409,10 @@ const onMapLoad = async () => {
           address: addressComponent, // driving directions in google, consider doing inside mapbox
           seekingMoney: (value, location) => needsMoneyComponent(location),
           seekingMoneyURL: (value, _) => '',
+          seekingVolunteers: (value, _) => sectionUrlComponent(value, 'seeking_volunteers'),
```

Comment:
❓ What other sections should this be done for? Based on my random clicking I saw these two sections as having lists but I'm sure there's a better way to identify this need 😅

Reply:
It's currently just volunteers and notes, and I don't foresee that changing unless (until?) we restructure/add columns... but I can make sure to let y'all know if that changes.

Reply:
Thank you for that information, @comeoneileen! :)